src/functionaltests/java/com/ericsson/ei/subscriptions/query/QueryAggregatedObjectsTestSteps.java

```diff
@@ -0,0 +1,229 @@
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.test.web.servlet.MvcResult;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.get; 
+
+import com.ericsson.ei.mongodbhandler.MongoDBHandler;
+import com.ericsson.ei.utils.FunctionalTestBase;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import cucumber.api.java.en.And;
+import cucumber.api.java.en.Given;
+import cucumber.api.java.en.Then;
+
+
+@Ignore
+@AutoConfigureMockMvc
+public class QueryAggregatedObjectsTestSteps extends FunctionalTestBase {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(QueryAggregatedObjectsTestSteps.class);
+
+    
+    private static final String aggregatedObjJsonPath = "src/test/resources/AggregatedObject.json";
+    private static final String missedNotificationJsonPath = "src/test/resources/MissedNotification.json";
+
+    
+    @Autowired
+    private MockMvc mockMvc;
+    private MvcResult mvcResult;
+    
+    @LocalServerPort
+    private int port;
+    
+    @Autowired
+    private MongoDBHandler mongoDBHandler;
+    
+    @Value("${spring.data.mongodb.database}")
+    private String eiDatabseName;
+    
+    @Value("${aggregated.collection.name}")
+    private String aggrCollectionName;
+    
+    @Value("${missedNotificationCollectionName}")
+    private String missedNotificationCollectionName;
+    
+    @Value("${missedNotificationDataBaseName}")
+    private String missedNotificationDatabaseName;
+
+    private String aggrObj;
+    private String missedNotificationObj;
+    
+    private ObjectMapper objMapper;
+    
+    public QueryAggregatedObjectsTestSteps() {
+        objMapper = new ObjectMapper();
+
+        try {
+            aggrObj = FileUtils.readFileToString(new File(aggregatedObjJsonPath), "UTF-8");
+            missedNotificationObj = FileUtils.readFileToString(new File(missedNotificationJsonPath), "UTF-8");
+
+        } catch (IOException e) {
+            LOGGER.error("Failed to open test json files for test. Error message\n: " + e.getMessage());
+            e.printStackTrace();
+        }
+    }
+    
+    @Given("^Aggregated object is created$")
+    public void aggregated_object_is_created() throws Throwable {
+        LOGGER.debug("Creating aggregated object in MongoDb");
+        assertEquals(true, createDocumentInMongoDb(eiDatabseName, aggrCollectionName, aggrObj));
+    }
+
+    @Given("^Missed Notification object is created$")
+    public void missed_notification_object_is_created() throws Throwable {
+        LOGGER.debug("Missed Notification object has been created in MongoDb");
+        assertEquals(true, createDocumentInMongoDb(missedNotificationDatabaseName, missedNotificationCollectionName, missedNotificationObj));
+    }
+
+    @Then("^Perform valid query on newly created Aggregated object")
+    public void perform_valid_query_on_newly_created_aggregated_object() throws Throwable {
```

Comment:
This method has become quite big, maybe som eparts of it could be moved out to it´s own method.